docs/docs/external/getting-started/guide.md

```diff
@@ -0,0 +1,28 @@
+---
+id: guide
+title: Using Weaver
+---
+
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+
+The easiest way to understand how Weaver works is to run it at a small scale:
+- First, [launch a set of basic test networks](./test-network/overview.md) built on Fabric and Corda. These networks offer the most basic capabilities of their DLT platforms and run toy applications (contracts and Layer-2) that can be easily tracked and debugged. You can launch these networks in one of several different ways: building Weaver components and dependencies locally or importing pre-built ones from Github packages, running core components in the host or in Docker containers. The choice depends on whether you just want to get these networks up and running or if you wish to customize the setup by modifying source code and configurations.
```

Comment:
Minor edits:
1. launch a set of basic test networks -> launch a set of test networks
2. run toy application -> run sample applications
3. that can be easily tracked -> that can easily be tracked

Reply:
Fixed 3. Need the words "basic" and "toy" to emphasize that these are very simple unrepresentative networks and apps we are talking about.